metal/mmtl/modules.py

```diff
@@ -16,6 +16,18 @@ def __init__(self, module):
     def forward(self, X):
         # The object that is passed out must be different from the object that gets
         # passed in so that cached outputs from intermediate modules aren't mutated
-        X_out = {k: v for k, v in X.items()}
-        X_out["data"] = self.module(X["data"])
-        return X_out
```

Comment:
In order to pass through the optional argument of token type ids to BERT, we need to modify this code so that the token  type is passed along with input ids. This is only done for the input to the BERT model, and not for the linear model at the end.